.travis.yml

```diff
@@ -1,21 +1,23 @@
 language: python
 
+sudo: false
+
 python:
   - "2.7"
   - "2.6"
+  - "3.4"
+
+addons:
+  apt:
+    packages:
+      - libgeos-c1
+      - libxml2-dev
 
 install:
-  - sudo apt-get install -y libgeos-c1
   - pip install . --use-mirrors
-  - pip install -r ./requirements-standalone.txt
-  - pip install -r ./requirements-dev.txt
-  - python ./setup.py -q install
-
-before_script:
-  - sudo add-apt-repository ppa:gcpp-kalxas/ppa-tzotsos -y
```

Comment:
Are any of the tests failing which causes OrderedDict to be required?

Sent from my iPhone

> On Sep 13, 2015, at 10:25, Filipe <notifications@github.com> wrote:
> 
> In .travis.yml:
> 
> >    - pip install . --use-mirrors
> > -  - pip install -r ./requirements-standalone.txt
> > -  - pip install -r ./requirements-dev.txt
> > -  - python ./setup.py -q install
> > -
> > -before_script:
> > -  - sudo add-apt-repository ppa:gcpp-kalxas/ppa-tzotsos -y
> I'm +1 with pushing the .travis.yml changes immediately in advance of this PR.
> 
> Makes sense. I actually should not have mixed things here. (Also, this PR might take a little more effort than I thought to make better use of six.moves and Ordereddicts.)
> 
> Done in #377
> 
> —
> Reply to this email directly or view it on GitHub.
>